models.py

```diff
@@ -108,7 +106,7 @@ def run(self):
 
     def countdown(self, exc_time):
         for sec in range(exc_time):
-            self.logger.info("{0} seconds until the next exercise".format(exc_time-sec))
+            self.logger.debug("{0} seconds until the next exercise".format(exc_time-sec))
```

Comment:
why set this to debug OOC? It should display things above info based on: https://github.com/dseisun/nerdouts/blob/master/models.py#L31